main.nf

```diff
@@ -501,7 +490,6 @@ process bowtie2 {
 
     input:
     file reads from trimmed_reads_bowtie2
-    set val(name) from name_for_bowtie2
 
     output:
     file '*.bowtie2.bam' into bowtie2_bam
```

Comment:
What does a typical filename look like from trim galore in this pipeline? Do you have an exact example? I would expect the input files to be something like `P1001_101_1.fastq.gz` and trim galore to generate something like `P1001_101_1_trimmed.fq.gz` - then you could just do `f=${f%_1_trimmed.fq.gz}`?